src/vm/lazy.ts

```diff
@@ -0,0 +1,31 @@
+import { clearError, processError } from '../error'
+import { status } from '../status'
+import { LazyPluginVM, PluginConfig, PluginsServer } from '../types'
+import { createPluginConfigVM } from './vm'
+
+export function createLazyPluginVM(
+    server: PluginsServer,
+    pluginConfig: PluginConfig,
+    indexJs: string,
+    logInfo = ''
+): LazyPluginVM {
+    const promise = createPluginConfigVM(server, pluginConfig, indexJs)
+        .then((vm) => {
+            status.info('🔌', `Loaded ${logInfo}`)
+            void clearError(server, pluginConfig)
+            return vm
+        })
+        .catch((error) => {
+            console.warn(`⚠️ Failed to load ${logInfo}`)
```

Comment:
Log via `status`?

Reply:
`status` does not have a warn method, copied the approach from previous code. Suggestions?

Reply:
Added `status.warn` method :)